Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditrepository/nhinc/AuditRepositoryOrchImpl.java

```diff
@@ -93,42 +93,32 @@ public AuditRepositoryOrchImpl() {
      */
     public AcknowledgementType logAudit(LogEventSecureRequestType mess, AssertionType assertion) {
 
-        AcknowledgementType response = null;
+        AcknowledgementType response = new AcknowledgementType();
         AuditRepositoryRecord auditRec = new AuditRepositoryRecord();
 
-        Date eventTimeStamp = null;
-        String eventCommunityId = null;
-
-        // This method should be removed when AuditLog.xsd schema changes are done. The values will be avialable from
-        //schema.
         EventIdentificationType eventIdentification = mess.getAuditMessage().getEventIdentification();
 
-        auditRec.setUserId("");
+        auditRec.setUserId(null);
 
-        // This method call should be removed when AuditLog.xsd schema changes are done. The values will be avialable
-        //from schema.
-        eventCommunityId = getCommunityID(mess);
+        String eventCommunityId = mess.getRemoteHCID();
         LOG.info("auditSourceID : " + eventCommunityId);
-        if (eventCommunityId != null && !eventCommunityId.equals("")) {
+        if (NullChecker.isNotNullish(eventCommunityId)) {
             auditRec.setRemoteHcid(eventCommunityId);
         } else {
             auditRec.setRemoteHcid("");
         }
 
-        auditRec.setDirection(mess.getInterface() + " " + mess.getDirection());
+        auditRec.setDirection(mess.getDirection());
         auditRec.setMessage(getBlobFromAuditMessage(mess.getAuditMessage()));
 
         XMLGregorianCalendar xMLCalDate = eventIdentification.getEventDateTime();
         if (xMLCalDate != null) {
-            eventTimeStamp = convertXMLGregorianCalendarToDate(xMLCalDate);
-            auditRec.setEventTimeStamp(eventTimeStamp);
+            auditRec.setEventTimeStamp(convertXMLGregorianCalendarToDate(xMLCalDate));
         }
 
-        // These are required DB data elements. But they should be cleaned up with appropriate values when
-        //AuditLog schema is updated
-        auditRec.setEventType("EventType");
+        auditRec.setEventType("servicetype");
         auditRec.setOutcome(0);
-        auditRec.setEventId("EventId");
+        auditRec.setEventId("evenId");
```

Comment:
Are "servicetype" (lack of camel-casing) and "evenId" (missing the "t" in event) correct here?